### DISCUSSION TITLE
Problema com o cadastro do GitHub Student

### DISCUSSION THREAD

**Discussion by jeffersonnjunior** [community/community#64944] [GitHub Education]
Estou enfrentando dificuldades ao tentar me cadastrar no GitHub Student. Para efetuar o registro, é necessário inserir o código enviado para o e-mail vinculado ao domínio da minha faculdade. Entretanto, a questão reside no fato de que este e-mail não possui autorização para utilizar o Outlook, o que impede o envio do código. Alguém tem outra alternativa?

**Comment by arafajs** [comment]
- Contact your school's IT department to see if they can help you get your school-issued email address authorized to use Outlook.
- Use a different email address that can receive emails from Outlook. This could be a personal email address, or an email address from a free service like Gmail or Yahoo Mail.
- Contact GitHub support and explain your situation. They may be able to help you register for GitHub Student without a school-issued email address.

Here are the steps on how to contact GitHub support:

1. Go to the GitHub Support: https://github.com/contact page.
2. Select the "I need help with my account" option.
3. Select the "I can't sign in" option.
4. Enter your GitHub username or email address.
5. Describe your issue in the text box.
6. Click the "Submit" button.

**Comment by jeffersonnjunior** [comment]
thank you very much
